bazel/testing/templates/golangci-lint.yaml

```diff
@@ -0,0 +1 @@
+
```

Comment:
I don't see bazel/testing/templates referenced anywhere? do we need those two files? Did I miss something?